cnd/src/seed.rs

```diff
@@ -85,4 +233,60 @@ mod tests {
         let debug = format!("{:?}", seed);
         assert_eq!(debug, "Seed([*****])".to_string());
     }
+
+    // 32 bytes base64 encoded.
+    const PEM: &str = "-----BEGIN SECRET SEED-----
+syl9wSYaruvgxg9P5Q1qkZaq5YkM6GvXkxe+VYrL/XM=
+-----END SECRET SEED-----
+";
+
+    #[test]
+    fn pem_library_works_as_expected() {
+        let pem = pem::parse(PEM).unwrap();
+        assert_eq!(pem.tag, "SECRET SEED");
+    }
+
+    #[test]
+    fn seed_from_pem_works() {
+        let want = base64::decode("syl9wSYaruvgxg9P5Q1qkZaq5YkM6GvXkxe+VYrL/XM=").unwrap();
```

Comment:
:question: Isn't it a little weird to have the secret seed hardcoded twice? Maybe we should have a constant for the secret seed, embed it in the PEM string and then compare the outcome of deriving the seed from the PEM string to the original seed? Not sure if this is better, just wondering.

Reply:
With the above mentioned test now removed these consts can be moved into the unit test that uses them.